app/assets/js/scripts/whitelist.js

```diff
@@ -3,24 +3,12 @@
 // switchView()
```

Comment:
moon2BOO

Reply:
Same as the first - I can comment in code why I left this here. We leverage this service but this is a case where the method is available on Global Scope because the wonky way this project works :eelsbadman: